websites/G/Genshin Impact Map/dist/metadata.json

```
@@ -6,7 +6,8 @@
 	},
 	"service": "Genshin Impact Map",
 	"description": {
-		"en": "Genshin Impact Interactive Map of all locations locations descriptions guides and more."
+		"en": "Genshin Impact Interactive Map of all locations locations descriptions guides and more.",
+		"nl": "Genshin Impact Interactieve kaart van alle locaties, locaties beschrijvingen gidsen en meer."
```

```suggestion
		"nl": "Genshin Impact's interactieve kaart met alle locaties, locatiebeschrijvingen, gidsen en meer."
```
This English description is a mess